scipy/stats/_stats_py.py

```diff
@@ -10675,6 +10675,148 @@ def first_order(t):
     return res.root
 
 
+def lmoments_iv(sample, order, axis, sorted, standardize):
```

Comment:
I think we need a docstring here, I'm having difficulty identifying this function's function and purpose in relation to `lmoments`-should this function be prefixed with `_`?

Reply:
"iv" in shorthand for "input validation (and standardization)" in several stats functions, but I can add a comment.

Reply:
Done

Reply:
Since it's It's a pretty straightforward private function and consistent with other `_iv` method in stats, I went ahead and added a comment rather than a full docstring.

> input validation/standardization for `lmoment`

The next commit will prepend the underscore, so resolving. Thanks!